benches/benches/bevy_reflect/map.rs

```diff
@@ -262,11 +262,11 @@ fn dynamic_map_insert(criterion: &mut Criterion) {
             &size,
             |bencher, &size| {
                 bencher.iter_batched(
-                    || DynamicMap::default(),
+                    DynamicMap::default,
                     |mut map| {
                         for i in 0..size as u64 {
                             let key = black_box(i);
-                            black_box(map.insert(key, i));
+                            map.insert(key, i);
```

Comment:
should map be blackboxed?

Reply:
black_box is a no-op according to hymm's comments, it should be deleted.

Reply:
`black_box(());` is a no-op, not sure about `black_box(map)` in this case

Reply:
Sorry I misunderstood.